templates/plan_template.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Template</title>
+    <!--Bootstrap CSS-->
+    <link rel="stylesheet" href="assets/css/styles.css">
```

Comment:
If we place the link of our self-defined styles.css after bootstrap.css, we should have the proper height of banner.
For reference you can check the order of <link> in search_page.html.

Reply:
Done